### ISSUE TITLE
AndroidManifest.xml's manifest tag package attribute is deprecated.

### ISSUE THREAD

**Issue by chrishpowell** [gluonhq/substrate#1188]
The following occurs during build/package process:
```
[Fri Nov 18 12:58:12 EET 2022][FINE] [SUB] Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
[Fri Nov 18 12:58:12 EET 2022][FINE] [SUB] Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
```

### Expected Behavior
The package entry in AndroidManifest manifest tag needs to move to an android{} definition in the build.gradle file so as the objection above no longer occurs.

### Current Behavior
The package name entry required in the AndroidManifest manifest tag causes the objection above to occur.

### Context
This is not a major issue at the moment but may have implications in the near future.

**Comment by javateer** [comment]
I started to see this after upgrading GRAALVM for my build environment.

But then I deleted **~/.gluon/substrate/Android** folder and let it recreate itself during the next build.

Also, I updated my src/android/AndroidManifest.xml file with attribute:

<activity android:name='com.gluonhq.helloandroid.MainActivity' **android:exported="true"** android:configChanges="orientation|keyboardHidden" android:screenOrientation="portrait">

Then this error message went away. If there's something else / more I did to clear it up, I'm not aware of it.

**Comment by jperedadnr** [comment]
fixed with #1251
